src/client/components/EventsPage/EventsPage.styles.css

```diff
@@ -0,0 +1,41 @@
+.all-container {
+  display: flex;
+  flex-direction: row;
+}
+
+.main-container {
+  border: solid 3px pink;
+  display: flex;
+  flex-direction: column;
+  background-color: #f8f8f8;
+  border-radius: 10px;
+  width: 30%;
+  padding: 14px 10px 10px 14px;
+}
+
+.top-part {
+  padding-left: 10px;
+  padding-bottom: 15px;
+}
+
+.logo-part {
+  display: flex;
+}
+
+.right-part {
+  margin-left: 10px;
+}
+
+.register-btn {
```

Comment:
There is already a `Button` component, can you use it instead of styling it yourself?